### PR TITLE
[16.0][FIX] pre-commit warnings

### PR DESCRIPTION
**Pull request by CRogos**
The branch 16.0 pre-commit job is not finishing without warnings

### CODE REVIEW

File: setup/_metapackage/VERSION.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-16.0.20230119.0
```
Comment (pedrobaeza) [inline setup/_metapackage/VERSION.txt]: Why are these files removed?

Comment (CRogos) [reply setup/_metapackage/VERSION.txt]: I just checked out branch 16.0 and executed pre-commit. 
Good question why it is removed?

Comment (CRogos) [reply setup/_metapackage/VERSION.txt]: I still don't know what is wrong with my other computer but on this these files are not deleted.

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking to remove warnings:

/ocabot merge patch
